jujugui/static/gui/src/app/components/unit-list/unit-list.js

```diff
@@ -55,6 +55,28 @@ YUI.add('unit-list', function() {
     },
 
     /**
+      The callable to be passed to the unit items for navigating to the unit
+      details.
+
+      @method _unitItemAction
+      @param {Object} e The click event.
+    */
+    _unitItemAction: function(e) {
+      var unitId = e.currentTarget.getElementsByTagName(
```

Comment:
the id should probably be put as a data attribute, that way the whole thing doesn't fail if we decide to change how the unit names are displayed.